examples/alarm_clock.py

```diff
@@ -268,7 +234,8 @@ def alarm_clock(robot):
     if alarm_time:
         print("Alarm set for %s" % alarm_time)
     else:
-        print("No Alarm time provided")
+        print("No Alarm time provided - usage e.g: 'alarm.clock.py 17:23' to set alarm for 5:23pm (input uses the 24 hour clock)")
```

Comment:
We could change this text as follows or similar:
"No Alarm time provided. Usage example: 'alarm.clock.py 17:23' to set alarm for 5:23 PM. (Input uses the 24-hour clock.)"